views/revert-modal.html

```diff
@@ -0,0 +1,53 @@
+{%- extends "apostrophe-modal:base.html" -%}
```

Comment:
Pretty sure this file is unnecessary and was part of a false start toward doing this with a modal that isn't needed?

Reply:
Correct. 
done